csvcubed/csvcubed/cli/entrypoint.py

```diff
@@ -105,4 +105,7 @@ def build_command(
 )
 def inspect_command(log_level: str, csvw_metadata_json_path: Path) -> None:
     start_logging(log_dir_name="csvcubed-cli", selected_logging_level=log_level)
-    inspect(csvw_metadata_json_path)
+    try:
+        inspect(csvw_metadata_json_path)
+    except Exception as e:
+        log_exception(_logger, e)
```

Comment:
logging makes sense but still put a `raise` on the next line (literally just `raise`, no argument, you get the full stack trace then). Otherwise while we are getting logs (fab) we're also swallowing the error, potentially quite confusing, if someones not checking logs it'd appear to run but do nothing.